modules/lti-service-impl/src/main/java/org/opencastproject/lti/service/impl/LtiServiceImpl.java

```diff
@@ -246,60 +250,84 @@ public void upsertEvent(
       throw new RuntimeException("No workflow configured, cannot upload");
     }
     try {
-      MediaPackage mp = ingestService.createMediaPackage();
+      final EventHttpServletRequest r = new EventHttpServletRequest();
```

Comment:
Please help those poor people who have to read a lot of code:
```suggestion
      final EventHttpServletRequest eventHttpRequest = new EventHttpServletRequest();
```